notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,817 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# LightGBM: A Highly Efficient Gradient Boosting Decision Tree\n",
+    "This notebook will give you a quick example of how to train LightGBM model in a recommendation scenario. \n",
```

Comment:
Can we provide more details here? "This notebook will give you a quick example of how to train a LightGBM model to estimate click-through rates on an e-commerce advertisement. We will train a LightGBM based model on the Criteo dataset."